doc/classes/RigidBody3D.xml

```diff
@@ -120,6 +120,14 @@
 				Sets an axis velocity. The velocity in the given vector axis will be set as the given vector length. This is useful for jumping behavior.
 			</description>
 		</method>
+		<method name="get_inverse_inertia_tensor">
+			<return type="Basis">
+			</return>
+			<description>
+				Returns the inverse inertia tensor basis. This is used to calculate the angular acceleration resulting from a torque applied to the RigidBody.
+			</description>
+
```

Comment:
There's an extraneous line here. Also, the methods are ordered alphabetically normally so it should be further up (you can run `./bin/godot.blabla --doctool .` to update the class reference automatically and fix it).